### PR TITLE
Bar chart tests

### PR DESCRIPTION
**Pull request by alvesjtiago**
Bar chart tests included mimicking previous line chart tests. Updated
ios-snapshot-test-case to 2.0.7 version.
Bar chart tests include:
- default chart
- hide values
- hide left axis
- hide right axis
- hide left axis gridlines
- hide right axis gridlines

### CODE REVIEW

File: Charts/ChartsTests/BarChartTests.swift [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        dataSet = BarChartDataSet(yVals: entries, label: "Bar chart unit test data")
+        
+        chart = BarChartView(frame: CGRectMake(0, 0, 480, 350))
+        chart.data = BarChartData(xVals: xValues, dataSet: dataSet)
+    }
+    
+    override func tearDown()
+    {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+        super.tearDown()
```
Comment (yas375) [inline Charts/ChartsTests/BarChartTests.swift]: Nit picking: no need to override `tearDown` here to just call super. Boiler plate code probably ;)

**Comment by yas375** [comment]
Cool! :+1:

**Comment by danielgindi** [comment]
Thanks :-)

File: Charts/ChartsTests/BarChartTests.swift (hunk 1 above)
Comment (alvesjtiago) [inline Charts/ChartsTests/BarChartTests.swift]: You're right @yas375, I just reused the code from the LineChartTests but I will refactor it to a single data generation file and keep the specifics to each of the tests. As well as removing the ```tearDown``` override :) thank you

**Pull request merged**